test/helpers/bytes32.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function bytes32(str) {
+    return web3.utils.padRight(web3.utils.toHex(str), 64);
```

Comment:
Can we use `ethers.formatBytes32String` instead?

Reply:
this would require adding ethers and anyway those helpers will disappear after we merge repos

Reply:
👍